### ISSUE TITLE
Change to web language detection breaks ListView

### ISSUE THREAD

**Issue by AlanSl** [nearform/polaris#62]
This is a bug that slipped through in one of the last few PRs. 

For some reason, `i18n.language` on web has started giving the full language for English (`en-US` or `en-GB`) whereas it used to return `en`. This is breaking the ListView page, which uses that code as an object key, causing it to get undefined values for translated text.

An easy quickfix would be to get the general form of the language similar to `src/services/i18n/index.native.js` does in native, maybe as a general `util` function:

```
  detect: callback => {
    // We will get back a string like "en-US".
    // We return a string like "en" to match our language files.
    callback(Localization.locale.split('-')[0]);
  },
```
...but it'd be good to know why this changed, and why some translations still work despite this change.

From reinstalling and running builds in different commits. I think it's due to a version change of some nested dependency in https://github.com/nearform/polaris/issues/50 but I haven't yet found what. On switching to commits since that one from older commits that work fine work, it works without running `npm i`, and fails after running `npm i`. 

However, obvious candidate, `i18next-browser-languagedetector`, appears to be at version 5.0.0 in both builds that pass and builds that fail.

**Comment by jh3y** [comment]
I couldn't replicate this when using `yarn`/`node` @AlanSl. On `14.0.0` currently.

**Comment by AlanSl** [comment]
@jh3y Ah, that's probably because https://github.com/nearform/polaris/pull/63 had been merged earlier that morning which contained a quickfix in the `ListView` view. So the question of why that became necessary (and why it worked before several seemingly unrelated PRs ago) is still open

**Comment by jh3y** [comment]
Without any steps to replicate the issue etc. It's impossible for anyone to know how to fix/if it's a fixable thing. 

Maybe a PR providing an `ISSUE_TEMPLATE.md` could avoid this 👍 🤔

**Comment by AlanSl** [comment]
Yeah it's a weird one. @jhey feel free to pick up something else (https://github.com/nearform/polaris/issues/68 might be a good one?)

**Comment by simoneb** [comment]
fixed in #111
